code/game/jobs/access.dm

```diff
@@ -253,20 +253,20 @@
 	switch(code)
 		if(0)

```

Comment:
please replace these with `#define`d values, such as `#define REGION_ALL 0`
there is only four uses of this proc anywhere